lib/iris/tests/test_analysis.py

```diff
@@ -365,6 +365,26 @@ def test_percentile_1d(self):
                        ('analysis', 'third_quartile_foo_1d.cml'),
                        checksum=False)
 
+    def test_fast_percentile_1d(self):
+        cube = tests.stock.simple_1d()
+
+        first_quartile = cube.collapsed('foo', iris.analysis.PERCENTILE,
+                                        percent=25, percentile_method='fast')
+        np.testing.assert_array_almost_equal(first_quartile.data,
+                                             np.array([2.5], dtype=np.float32))
+        self.assertCML(first_quartile, ('analysis',
```

Comment:
This may not be what you were after, but it is a bit tidier. Feel free to suggest further changes, or the return of the CML statements as you prefer.